xml/System/DateTime.xml

```diff
@@ -8102,7 +8102,7 @@ The value of the current <xref:System.DateTime> object is formatted using the pa
       <Docs>
         <param name="s">A string containing a date and time to convert.</param>
         <param name="result">When this method returns, contains the <see cref="T:System.DateTime" /> value equivalent to the date and time contained in <paramref name="s" />, if the conversion succeeded, or <see cref="F:System.DateTime.MinValue" /> if the conversion failed. The conversion fails if the <paramref name="s" /> parameter is <see langword="null" />, is an empty string (""), or does not contain a valid string representation of a date and time. This parameter is passed uninitialized.</param>
-        <summary>To be added.</summary>
+        <summary>Converts the specified string representation of a date and time to its <see cref="T:System.DateTime" /> equivalent and returns a value that indicates whether the conversion succeeded.</summary>
```

Comment:
>string [](start = 40, length = 6)

this should be a "char span"

Reply:
isn't char span a string representation?

Reply:
mentioning char span help when reading the doc to not go back and check the type.